storage/feed.go

```diff
@@ -79,6 +79,34 @@ type StartStoreEvent struct {
 	StoreID proto.StoreID
 }
 
+// StoreStatusEvent contains the current descriptor for the given store.
+//
+// Because the descriptor contains information that cannot currently be computed
+// from other events, this event should be periodically broadcast by the store
+// independently of other operations.
+type StoreStatusEvent struct {
+	StoreID proto.StoreID
+	Desc    *proto.StoreDescriptor
```

Comment:
what do you mean by "change"? is it being updated in place or replaced entirely?

if it's the former, all the pointers will point to the same descriptor anyway, so i don't see why that'd preclude you from putting the pointer on the `StoreEventFeed`.